.travis.yml

```diff
@@ -0,0 +1,178 @@
+#  - sudo echo "FileUsage = 1" >> odbcinst.ini
+  #- ls -al /etc/odbcinst.ini /etc/odbc.ini
+  - cat /etc/odbcinst.ini
+  - cat  /etc/odbc.ini
+ # - ls /etc/ODBCDataSources
+ # - cat  .odbc.ini #not found
+  #
+  - echo "[global]" > .freetds.conf
+  - echo "tds version = 4.2" >> .freetds.conf
+  - echo "[kyle]" >> .freetds.conf
+  - echo "host = nrb8xkgxaj.database.windows.net" >> .freetds.conf
+  - echo "port = 1433" >> .freetds.conf
+  - echo "database=odm2" >> .freetds.conf
+  - echo "[moonstone]">> .freetds.conf
+  - echo "host = moonstone.sdsc.edu">> .freetds.conf
+  - echo "port = 1433">> .freetds.conf
+  - echo "database=odm2" >> .freetds.conf
+  - cat .freetds.conf
+  - sudo apt-get install mysql-client
+  # Spatialiate
+  - sudo apt-get install -y libproj-dev libgeos-dev libspatialite-dev
+  - sudo ln -s /usr/lib/x86_64-linux-gnu/libspatialite.so /usr/lib/libspatialite.so
+#  - sudo apt-get install python-scipy python-matplotlib python-pandas python-sympy python-nose
+  - sudo apt-get install python-matplotlib python-pandas python-nose
+  - python -m pip install pandas
+  - export PYMSSQL_BUILD_WITH_BUNDLED_FREETDS=1;python -m  pip install pymssql
+  - python -m  pip install mysql-python
+install: # now just our code
+  - pip install git+https://github.com/ODM2/geoalchemy.git@odm2#egg=geoalchemy-0.7.3
+  - pip install   .
+  - pip install  -r requirements_tests.txt  --allow-external pyodbc --allow-unverified pyodbc
+    # pysqlite
+  - pip install pysqlite
+  - pip list
+  - ls -al  ./tests/usecasesql/littlebearriver/sampledatabases/odm2_mysql/LBR_MySQL_SmallExample.sql ./tests/usecasesql/marchantariats/marchantariats.sql
+  - mysql --user=ODM --password=odm  odm2 < ./tests/usecasesql/littlebearriver/sampledatabases/odm2_mysql/LBR_MySQL_SmallExample.sql
+  - mysql --user=root  -e  "show databases;"
+  - mysql --user=root  -e "select table_schema,table_name,table_rows from information_schema.tables;"
+  - mysql --user=root -e   "GRANT ALL PRIVILEGES ON odm2.* TO 'ODM'@'localhost';FLUSH PRIVILEGES;"
+  - mysql --user=ODM --password=odm odm2 -e "use odm2; Select * from Variables;"
+  - mysql --user=ODM --password=odm odm2 -e "use odm2; Select * from variables;"
+#  - mysql --user=ODM --password=odm odm2 -e "Select * from odm2.Variables;"
+  - mysql --user=ODM --password=odm  -e "Select * from odm2.Variables;"
+  - mysql --user=ODM --password=odm  -e "Select * from odm2.variables;"
+# add -a to psql to see full log, -q is quiet
+  - psql -U postgres -q  -f ./tests/usecasesql/marchantariats/marchantariats.sql
+#  - netstat output | grep mysql
+  - mysql --user root -e   "SELECT User, Host FROM mysql.user;"
+
+# don't forget to open up the azure mssql server to these addreses
+# https://docs.travis-ci.com/user/ip-addresses/
+
+# command to run tests
+script:
+# just the connection part
+  - py.test tests/test_SessionFactory.py
+  - py.test tests/test_connection.py
+  - py.test
```

Comment:
We can create a test matrix to start these 3 tests at the same time and separate the context if you think it is worth it.